python/cugraph/cugraph/structure/renumber_wrapper.pyx

```diff
@@ -49,14 +49,20 @@ cdef renumber_helper(shuffled_vertices_t* ptr_maj_min_w, vertex_t, weights):
         move(pair_s_minor.first), vertex_t, "shuffled_minor")
 
     shuffled_df = cudf.DataFrame()
+    # Some workers might have no data therefore ensure the empty column have the appropriate
+    # vertex_t or weight_t. Failing to do that will create am empty column of type object
+    # which is not supported by '__cuda_array_interface__'
     shuffled_df['major_vertices']=shuffled_major_series
+    shuffled_df['major_vertices']=shuffled_df['major_vertices'].astype(vertex_t)
```

Comment:
I guess you were not able to just add the "astype" to the series?   What is the performance hit of doing an "astype"?

Reply:
Performance wised, it is not too bad because this operation mainly target to change the type of a `None` column to the appropriate dtype.  As this PR stated, some workers end up with no data after shuffling hence `shuffled_major_series` end up being None which translate to a column of type object(causing the error). Regarding the case where `shuffled_major_series` is a series with data, it will already have the appropriate dtype hence I will be doing a cast of the same dtype which takes 10^-3 seconds for a series where the dataset is of scale 24. I can add a condition to only cast when `shuffled_major_series` is None

Reply:
Addressed

Reply:
@BradReesWork regarding this comment (I didn't see a better way to respond to your last comments)

> any insight into why tests are failing (gpuCI-cuda11.5/cugraph/gpu/cuda/11.5/python/3.8/centos7 )?

A lot of the 11.5 tests are failing but I though we could still merge PR for now though. I will look into this and see what's going on